styles.css

```diff
@@ -138,7 +142,7 @@ h1 {
 }
 
 .game-button {
-  background-color: #d0d2d3;
```

Comment:
Changed background color to be darker and added a disabled class that will be the lighter color naturally until the appropriate input has content and then the disabled class will be removed